config/kms/config.go

```diff
@@ -33,6 +33,13 @@ func Configure(p *config.Provider) {
 		}
 	})
 
+	p.AddResourceConfigurator("aws_efs_file_system", func(r *config.Resource) {
+		r.References["kms_key_id"] = config.Reference{
+			Type:      "github.com/upbound/provider-aws/apis/kms/v1beta1.Key",
+			Extractor: common.PathARNExtractor,
+		}
+	})
```

Comment:
Could you please move this config to https://github.com/upbound/provider-aws/blob/main/config/efs/config.go as the main resource is in the efs group?

Reply:
Makes sense, done.

Tested again and it looks good

```
NAME                                                  READY   SYNCED   EXTERNAL-NAME          AGE
filesystem.efs.aws.upbound.io/example-efs-encrypted   True    True     fs-040c785a941xxxxxx   12m

NAME                                 READY   SYNCED   EXTERNAL-NAME                          AGE
key.kms.aws.upbound.io/example-key   True    True     d4b0e1dc-7795-4631-be1f-xxxxxxxxxxxx   12m
```